src/bundle.js

```diff
@@ -9,9 +9,14 @@ module.exports = function (packagePath) {
     return getVendors(entries, packagePath)
       .then(extractPackageJsonPaths(entries, packagePath))
       .then(function (results) {
+        // Grab all extra entries related to browser mappings
+        var mapEntries = Object.keys(entries).reduce(function (currentMapEntries, entryKey) {
```

Comment:
Only browser mapping is put into this mapping now, but could be more later I suppose :)